### PR TITLE
Update parallel computing docs regarding configuration

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR removes section of text in doc/guides/parallel_computing.rst regarding the outdated NEST_SOURCE_DIR/configure --with-mpi and replaces it with a brief description of using the -Dwith-mpi=ON 

Additionally, some minor fixes to broken links and changes to the text were made to improve clarity.

### CODE REVIEW

File: doc/guides/parallel_computing.rst [hunk 1]
```diff
@@ -1,213 +1,227 @@
...
+* :ref:`Distributed simulation (or distributed computing) <distributed_computing>`
+     * uses the Message Passing Interface (MPI)
+     * supports simulations over multiple computers
+
+Both methods can be combined within a simulation.
+
+
+See `Plesser et al. (2007) <http://dx.doi.org/10.1007/978-3-540-74466-5_71>`__
+for more information on NEST parallelization and be sure to check the
 documentation on `Random numbers in NEST <random-numbers.md>`__.
```
Comment (Silmathoron) [inline doc/guides/parallel_computing.rst]:
```suggestion
documentation on :ref:`Random numbers in NEST <random_numbers>`.
```
Then please add ``.. _random_numbers:`` at the top of ``random_numbers.rst``.

File: doc/guides/parallel_computing.rst [hunk 2]
```diff
@@ -1,213 +1,227 @@
...
+
+Neuron distribution
+~~~~~~~~~~~~~~~~~~~~
+
+Neurons are assigned to one of the virtual processes in a round-robin fashion.
+On all other virtual processes, no object is created. Proxies ensure the id
+of the real node on a given VP is kept free.
+
+The virtual process `idVP` on which a neuron with global id `idNode` is
+allocated is given by :math:`idVP = idNode %NVP`, where `NVP` is the total
```
Comment (Silmathoron) [inline doc/guides/parallel_computing.rst]:
```suggestion
allocated is given by :math:`id_{vp} = gid_{node} %N_{vp}`, where :math:`N_{vp}` is the total
```

File: doc/guides/parallel_computing.rst [hunk 3]
```diff
@@ -1,213 +1,227 @@
...
-*to\_file* set to *true*), the distribution also results in multiple
+
+Neuron distribution
+~~~~~~~~~~~~~~~~~~~~
+
+Neurons are assigned to one of the virtual processes in a round-robin fashion.
+On all other virtual processes, no object is created. Proxies ensure the id
+of the real node on a given VP is kept free.
+
+The virtual process `idVP` on which a neuron with global id `idNode` is
```
Comment (Silmathoron) [inline doc/guides/parallel_computing.rst]:
```suggestion
The virtual process :math:`id_{vp}` on which a neuron with global id :math:`gid_{node}` is
```

File: doc/guides/parallel_computing.rst [hunk 4]
```diff
@@ -1,213 +1,227 @@
...
+The first part is the name of the `model` (e.g. ``voltmeter`` or
+``spike_detector``) or, if set, the `label` of the recording device. Next is
+the global id (GID) of the recording device, followed by the id of the VP
+assigned to the recorder. Spike files have the file extension ``gdf`` and
+analog recordings from the ``multimeter`` have ``dat`` as file extension.
+
+The ``label`` and ``file_extension`` of a recording device can be set like any
+other parameter of a node using ``SetStatus``.
+
+.. note::
```
Comment (Silmathoron) [inline doc/guides/parallel_computing.rst]: Please move this note between Figure 13 and "Neuron distribution"

File: doc/guides/parallel_computing.rst [hunk 5]
```diff
@@ -1,213 +1,227 @@
...
+
+ The status dictionary of each node (i.e. neuron or device) contains
+ three entries that are related to parallel computing:
 
-Spike exchange and synapse updates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *  *local* (boolean): indicating if the node exists on the local process or not
+ *  *thread* (integer): id of the local thread the node is assigned to
+ *  *vp* (integer): id of the virtual process the node is assigned to
```
Comment (Silmathoron) [inline doc/guides/parallel_computing.rst]:
```suggestion
 *  *vp* (integer): id of the virtual process the node is assigned to (denoted :math:`id_{vp}` here)
```

**Comment by Silmathoron** [comment]
I discussed with @jougs and he says you can proceed with the changes

**Comment by terhorstd** [comment]
Nice work! That was really quick!

**Pull request merged**